benchmark/Streamly/Benchmark/FileSystem/Handle.hs

```diff
@@ -65,7 +65,7 @@ blockCount size = (size + blockSize - 1) `div` blockSize
 main :: IO ()
 main = do
     (_, cfg, benches) <- parseCLIOpts defaultStreamSize
-    r <- lookupEnv "Benchmark.FileSystem.Handle.InputFile"
+    r <- lookupEnv "Benchmark_FileSystem_Handle_InputFile"
```

Comment:
Where do we set this variable?

Reply:
In the shell when running the benchmark.